app/views/admin/dashboard/show.html.haml

```diff
@@ -35,7 +35,7 @@
                 = link_to event_race_participants_path(race.event, race), target: "_blank" do
                   %i.fa.fa-external-link-alt{"data-placement" => "right", "data-toggle" => "tooltip", :title => "See the start list"}
                   = race.name_with_event
-              %td= race.paid_participants_count
+              %td= race.participants.for_start_list.size
               %td= race.participants_cap
-              %td= race.participants_cap - race.paid_participants_count
+              %td= race.participants_cap - race.participants.for_start_list.size
```

Comment:
This is the part of this PR that _fixes_ -- the rest seems to be additive. Split the two up! It's a great way to learn to isolate your changes to that which is most pertinent to the issue being resolved.

Reply:
I had this thought after I created the PR. I'll split them up, I knew better 😔